src/Features/LanguageServer/Protocol/Features/Diagnostics/DiagnosticAnalyzerService_IncrementalAnalyzer.cs

```diff
@@ -6,58 +6,26 @@
 using Microsoft.CodeAnalysis.Diagnostics.EngineV2;
 using Microsoft.CodeAnalysis.Host.Mef;
 using Microsoft.CodeAnalysis.Internal.Log;
-using Microsoft.CodeAnalysis.SolutionCrawler;
 using Roslyn.Utilities;
 
-namespace Microsoft.CodeAnalysis.Diagnostics
+namespace Microsoft.CodeAnalysis.Diagnostics;
+
+internal partial class DiagnosticAnalyzerService
 {
-    [ExportIncrementalAnalyzerProvider(
-        name: WellKnownSolutionCrawlerAnalyzers.Diagnostic,
-        workspaceKinds: [WorkspaceKind.Host, WorkspaceKind.Interactive],
-        highPriorityForActiveFile: true)]
```

Comment:
note: the entire system for exporting/importing incremental analyzers went away.  This was hte only incremental analyzer left, and we don't have solution crawler anymore, so it wasn't needed.  Note that thsi type is *also* exported as an IDiagnosticAnalyzerService which is *instead* the api that features should use.